app/controllers/api/service_templates_controller.rb

```diff
@@ -52,5 +53,9 @@ def unarchive_resource(type, id, _data)
     def set_additional_attributes
       @additional_attributes = %w(config_info)
     end
+
+    def encode_picture(data)
```

Comment:
Should this be `decode_picture` instead?

Reply:
yeah, names n thing n stuff n whatever